packages/slice-machine/pages/onboarding.tsx

```diff
@@ -140,6 +140,40 @@ const StepIndicator = ({
   );
 };
 
+function handleTracking(props: {
+  step: number;
+  startTime: number;
+  maxSteps: number;
+}) {
+  const state = useRef(props);
+
+  useEffect(() => {
+    state.current = props;
+  }, [props]);
+
+  useEffect(
+    () => () => {
+      const { startTime, maxSteps, step } = state.current;
+      const endTime = Date.now();
+      const totalTime = endTime - startTime;
+      const data = {
+        lastStep: step,
+        maxSteps,
+        startTime,
+        endTime,
+        totalTime,
+      };
+
+      fetch("/tracking/onboarding", {
```

Comment:
Isn't there a module with these API calls ? (I really don't know that's why i'm asking).

Reply:
tbh, I don't know if there is